unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  if (!Array.isArray(numbers)) {
+    return 'Please provide an array.';
+  }
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ * It works for both integer and fractional numbers.
+ * That is, `toDollars(12)` returns `$12.00` and
+ * `toDollars(11.341)` returns `$11.34`.
+ */
+export function toDollars(amount) {
+  if (Number.isNaN(Number(amount))) {
```

Comment:
Nice edge case check. See above.

You have many very nice edge case checks, but they should all throw.